guides/common/assembly_preparing-environment-for-installation-in-ipv6-network.adoc

```diff
@@ -0,0 +1,11 @@
+[id="preparing-environment-for-installation-in-ipv6-network"]
+= Preparing your Environment for {Project} Installation in an IPv6 Network
+
+To provision hosts in an IPv6 network, you must install {Project} in an IPv6 network and configure {Project} for the UEFI HTTP boot provisioning. Before installing {Project} in an IPv6 network, view the limitations and ensure that you meet the requirements.
```

Comment:
I'd probably break up these two requirements because the first one is required always, however UEFI HTTP boot provisioning is only required when you plan to do network boot provisioning.